app/utils/exists.js

```diff
@@ -0,0 +1,4 @@
+export default function exists(selector) {
+  const el = document.querySelector(selector);
+  return !!el;
```

Comment:
This can be one line: `return !!document.querySelector(selector);` Which begs the question of needing it to be it's own function.

Reply:
This was actually to help keep track of the back and forth logic between when to show and hide certain disclaimers. I was also able to reuse in `post-search-results.js`.

Reply:
Looks cleaner: 325d13f